app/views/users/search.js.erb

```diff
@@ -1,6 +1,26 @@
 
 <% if @results.any? %>
-  $('#user-list').html("<%= escape_javascript(render partial: 'users/users_table', locals: { users: @results }) %>");
+  $('#user-list').html("<%= escape_javascript(render partial: 'users/users_table', locals: { users: @results, autosortable: true }) %>");
+
+  //TODO figure out way to import function from /javascripts/table_sorting.coffee
```

Comment:
So I checked and I think this is a problem in table content on the dashboard after we add or remove a user, too. I wonder if we can move the stupidtable invocation to its own partial (something like `app/views/shared/_activate_stupidtable.js.erb`) and drop in something like `<%= render partial: shared/activate_stupidtable %>`?

Reply:
Don't know where the edge cases are but I made the partial for rendering.